File: test-tabs.py
import asyncio
from playwright.async_api import async_playwright

async def test_tabs():
    async with async_playwright() as p:
        browser = await p.chromium.launch()
        page = await browser.new_page(viewport={"width": 1440, "height": 900})
        
        await page.goto("https://deepreading.qzz.io", wait_until="networkidle")
        await page.wait_for_timeout(2000)
        
        # Tab 0: 文献筛选 (already active)
        await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_0_filter.png")
        print("Tab 0 (filter) OK")
        
        # Tab 1: 长文本精读
        tab1 = await page.get_by_text("长文本精读").first
        if tab1:
            await tab1.click()
            await page.wait_for_timeout(2000)
            await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_1_long.png")
            print("Tab 1 (long) OK")
        
        # Tab 2: 七步精读
        tab2 = await page.get_by_text("七步精读").first
        if tab2:
            await tab2.click()
            await page.wait_for_timeout(2000)
            await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_2_quant.png")
            print("Tab 2 (quant) OK")
        
        # Tab 3: 四步精读
        tab3 = await page.get_by_text("四步精读").first
        if tab3:
            await tab3.click()
            await page.wait_for_timeout(2000)
            await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_3_qual.png")
            print("Tab 3 (qual) OK")
        
        # Tab 4: 提示词管理
        tab4 = await page.get_by_text("提示词管理").first
        if tab4:
            await tab4.click()
            await page.wait_for_timeout(2000)
            await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_4_prompts.png")
            print("Tab 4 (prompts) OK")
        
        # Back to Tab 0
        tab0 = await page.get_by_text("文献筛选").first
        if tab0:
            await tab0.click()
            await page.wait_for_timeout(2000)
            await page.screenshot(path="/root/.openclaw/workspace/deep-reading-agent/tab_0_back.png")
            print("Tab 0 back OK")
        
        await browser.close()
        print("All tabs tested successfully")

asyncio.run(test_tabs())

File: test-tabs.js
// test-tabs.js - 测试 Tab 切换
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage({ viewport: { width: 1440, height: 900 } });
  
  await page.goto('https://deepreading.qzz.io', { waitUntil: 'networkidle' });
  await page.waitForTimeout(2000);
  
  // Screenshot 1: 首页（文献筛选 Tab）
  await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_0_filter.png' });
  console.log('Tab 0 (filter) OK');
  
  // Click Tab 1: 长文本精读
  const tab1 = await page.$('text=长文本精读');
  if (tab1) {
    await tab1.click();
    await page.waitForTimeout(2000);
    await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_1_long.png' });
    console.log('Tab 1 (long) OK');
  }
  
  // Click Tab 2: 七步精读
  const tab2 = await page.$('text=七步精读');
  if (tab2) {
    await tab2.click();
    await page.waitForTimeout(2000);
    await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_2_quant.png' });
    console.log('Tab 2 (quant) OK');
  }
  
  // Click Tab 3: 四步精读
  const tab3 = await page.$('text=四步精读');
  if (tab3) {
    await tab3.click();
    await page.waitForTimeout(2000);
    await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_3_qual.png' });
    console.log('Tab 3 (qual) OK');
  }
  
  // Click Tab 4: 提示词管理
  const tab4 = await page.$('text=提示词管理');
  if (tab4) {
    await tab4.click();
    await page.waitForTimeout(2000);
    await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_4_prompts.png' });
    console.log('Tab 4 (prompts) OK');
  }
  
  // Back to Tab 0
  const tab0 = await page.$('text=文献筛选');
  if (tab0) {
    await tab0.click();
    await page.waitForTimeout(2000);
    await page.screenshot({ path: '/root/.openclaw/workspace/deep-reading-agent/tab_0_back.png' });
    console.log('Tab 0 back OK');
  }
  
  await browser.close();
  console.log('All tabs tested successfully');
})();
